src/familytoolswap_backend/routes.py

```diff
@@ -93,9 +93,23 @@ def twotables():
 
 #@app.route("/customer/history/year/<year>", methods=["GET"])
 
-@app.route("/tool_search/<desc>")
-def tool_search(desc):
+@app.route("/tool_search/<description>/<type>")
+def tool_search(description, type):
+	# tool_search_sql = "SELECT id, name, description FROM tool_inventory JOIN owner ON tool_inventory.id = owner.owner_id WHERE description = '" + description +"'"
+	# tool_search_sql = f"SELECT id, name, description FROM tool_inventory JOIN owner ON tool_inventory.id = owner.owner_id WHERE description = '{ description }'"
+	tool_search_sql = "SELECT id, name, description, type FROM tool_inventory JOIN owner ON tool_inventory.id = owner.owner_id WHERE description = %s and type = %s"
```

Comment:
Advanced tip:
If your SQL is getting really long you can do something like this:
```
tool_search_sql = "SELECT id, name, description, type" +
  " FROM tool_inventory" +
  " JOIN owner ON tool_inventory.id = owner.owner_id" +
  " WHERE description = %s and type = %s"
```
making sure that there is a space *inside* the double quotes. If you do this:
```
tool_search_sql = "SELECT id, name, description, type" +
  "FROM tool_inventory" +
  "JOIN owner ON tool_inventory.id = owner.owner_id" +
  "WHERE description = %s and type = %s"
```
then this will result in `"SELECT id, name, description, typeFROM tool_inventoryJOIN owner ON tool_inventory.id = owner.owner_idWHERE description = %s and type = %s"`, which is clearly not what you want.